src/timepicker/timepicker.js

```diff
@@ -160,7 +160,9 @@ class TimePicker extends React.Component<TimePickerPropsT, TimePickerStateT> {
 
   handleChange = (seconds: number) => {
     const date =
-      this.props.value && seconds ? new Date(this.props.value) : new Date();
+      this.props.value && typeof seconds === 'number'
```

Comment:
Based on the type definition, it needs to be a number or flow will error. In what situation will seconds not be defined?

Reply:
seconds can be zero, making it a falsy value - this change tries to cover that case

Reply:
can `&& seconds` be removed?

Reply:
that's a very good point - yes

Reply:
@chasestarr fixed - @nadiia does this make sense to you too?